docs/ppl/datatypes.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Data Types
+parent: Piped Processing Language
+nav_order: 6
+---
+
+
+# Data Types
```

Comment:
Sentence case for headers in the ODFE docs.

Reply:
Fixed.